scripts/lib/install

```diff
@@ -689,12 +699,6 @@ EOF
     exit 0
 fi
 
-# Frontend deploys use /home/zulip/deployments; without this, the
-# install directory is also only readable by root.
-deploy_path=$("$ZULIP_PATH"/scripts/lib/zulip_tools.py make_deploy_path)
-mv "$ZULIP_PATH" "$deploy_path"
-ln -nsf /home/zulip/deployments/next "$ZULIP_PATH"
-ln -nsf "$deploy_path" /home/zulip/deployments/next
 ln -nsf "$deploy_path" /home/zulip/deployments/current
```

Comment:
It seems odd to leave this symlink down here, and not above when we set up `next`

Reply:
Well the creation of `next` needed to move to avoid leaving a broken symlink if we fail somewhere between there and here. But the point where we start using `current` is still here.

The original intention of 92a230acaa506b7252571670f232790e81a1ca51 was that linking `current` is the last step of installation, because we aren’t “current” until we’re installed. Since then several steps were tacked on afterwards, as things are. But we’re still pretty close to the last point before a possible early `exit`.